templates/preempt-rt/preempt-rt.jinja2

```diff
@@ -0,0 +1,30 @@
+- test:
+    timeout:
+      minutes: 10
+
+    definitions:
+    - from: inline
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: preempt-rt-prereq
+          description: Pre-requisites for PREEMPT_RT
+        run:
+          steps:
+          - apt-get update && apt-get install -y procps rt-tests
```

Comment:
Looks like something to add to the rootfs image as a follow-up - either `buster-rt` if we want to keep adding test suites for real-time, if not this could be added to the basic `buster` one to avoid rootfs proliferation.

Reply:
Yes, this can be done if/when creating a test-plan specific rootfs.